src/Command/BaseCommand.php

```diff
@@ -27,6 +45,8 @@ class BaseCommand extends Command
     public function __construct(string $name = null)
     {
         parent::__construct($name);
+        $this->config = $this->getConfig();
+        $this->drushRunner = new DrushCommandRunner();
```

Comment:
Keeping this for now, but making an issue for this to be revisited / refactored once WP support comes into play.

Reply:
Followup Issue: https://github.com/daceej/waffle/issues/35